examples/riot/external_modules/reactor-uc/Makefile.include

```diff
@@ -0,0 +1,9 @@
+
+CFLAGS += -DPLATFORM_RIOT=1
+
+# Use an immediate variable to evaluate `MAKEFILE_LIST` now
+USEMODULE_INCLUDES_reactor-uc := $(LAST_MAKEFILEDIR)/../../../../include
+USEMODULE_INCLUDES += $(USEMODULE_INCLUDES_reactor-uc)
+
+USEMODULE_INCLUDES_reactor-uc2 := $(LAST_MAKEFILEDIR)/../../../../external
```

Comment:
Regarding this thing I have no Idea why I needed to do it like this, but it works for now. I will ask some people who know the riot build system better to improve this in the future ..